services/web-app/components/mdx-notification/mdx-notification.module.scss

```
@@ -0,0 +1,115 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@use '@carbon/react/scss/spacing' as spacing;
+@use '@carbon/react/scss/type' as type;
+@use '@carbon/react/scss/breakpoint' as breakpoint;
+@use '@carbon/react/scss/theme' as theme;
+@use '@carbon/react/scss/utilities/convert' as convert;
+
+// override carbon low contrast anchor
+:global(.cds--inline-notification:not(.cds--inline-notification--low-contrast) a) {
+  color: theme.$link-primary;
+}
+
+// override carbon tooltip
+:global(.cds--icon-tooltip) {
+  position: absolute;
+  top: 1px;
+  right: 1px;
+}
+
+// override carbon - icon inline with title
+:global(.cds--inline-notification__icon) {
+  margin-top: spacing.$spacing-05;
+}
+
+// override carbon max-width
+:global(.cds--inline-notification) {
+  max-width: 100%;
+}
+
+.error-container {
+  position: relative;
+  margin-top: spacing.$spacing-05;
+}
```

I'd suggest scoping these global styles only to this component to be safe. 

```suggestion

.error-container {
  position: relative;
  margin-top: spacing.$spacing-05;

  // override carbon low contrast anchor
  :global(.cds--inline-notification:not(.cds--inline-notification--low-contrast) a) {
    color: theme.$link-primary;
  }

  // override carbon tooltip
  :global(.cds--icon-tooltip) {
    position: absolute;
    top: 1px;
    right: 1px;
  }

  // override carbon - icon inline with title
  :global(.cds--inline-notification__icon) {
    margin-top: spacing.$spacing-05;
  }

  // override carbon max-width
  :global(.cds--inline-notification) {
    max-width: 100%;
  }
}
```